dev-docs/source/Testing/ErrorReporter-ProjectRecovery/ErrorReporterTesting.rst

```diff
@@ -0,0 +1,100 @@
+.. _error_reporter_testing:
+
+Error Reporter Testing
+======================
+
+.. contents::
+  :local:
+
+Error Reporter test
+-------------------
+
+*Preparation*
+
+-  Ensure that `Report usage data` is checked in the First Time Setup screen
+-  Files ``SANS2D00005512.nxs``
```

Comment:
May be worth pointing out here that this file can be found in Testing/Data/SystemTest/SANS2D in the source directory or ExternalData/Testing/Data/SystemTest/SANS2D in the build directory. I think this is fine for now as a method of provoking an unhanded exception.